hibernate-core/src/main/java/org/hibernate/query/spi/AbstractQuery.java

```diff
@@ -1403,8 +1403,7 @@ else if ( retType.isArray() ) {
 	}
 
 	@Override
-	@SuppressWarnings( "rawtypes" )
-	public QueryImplementor<R> setProperties(Map map) {
+	public QueryImplementor<R> setProperties(Map<?,?> map) {
```

Comment:
I always think inserting a space char between `?,?` improves code readability (and in line with our existing coding format). 
```
public QueryImplementor<R> setProperties(Map<?, ?> map) {
... ...
}
```
I always hesitated whether nailing down the first type generics to String is better or not
```
public QueryImplementor<R> setProperties(Map<String, ?> map) {
```

Reply:
While we don't have a specific code guideline around this, I personally prefer the no space version as well until these get "nested".

But I'm certainly not going to ask someone to make that change in a PR